packages/medusa-test-utils/package.json

```diff
@@ -42,6 +43,9 @@
     "@medusajs/medusa": {
       "optional": true
     },
+    "@medusajs/modules-sdk": {
+      "optional": true
+    },
```

Comment:
thought: looking at the code this doesn't seem optional. E.g., 

```
const moduleSdkImports = require("@medusajs/modules-sdk")
```

Will fail if there is no @medusajs/modules-sdk installed so isn't it more correct to leave it as a required peer dep?

Reply:
Yes my bad, i wanted to put it as a peerDependencies, i was playing with it to see if one of my issue was coming from the dep or something else and then forgot to move it back